README.md

```diff
@@ -120,10 +119,15 @@ WORKSPACE, and aliases to the explicit dependencies. Slightly different plumbing
 is required.
 
 #### Generate a Cargo.toml
-Generate a Cargo.toml, similar to Vendoring mode but add a new directive in the
+Generate a Cargo.toml, similar to Vendoring mode but add a new `genmode` directive in the
 `[raze]` section
 ```toml
-[raze]
```

Comment:
We'd probably want to leave this `[raze]` section header in the example to provide a little better reference with the previous example.

Reply:
Whoops, fixed.

Reply:
Thanks